Lacework/lacework_module/metrics.py

```diff
@@ -0,0 +1,33 @@
+from prometheus_client import Counter, Histogram
```

Comment:
The infrastructure suggest us to use a Gauge for the `event_lags` metric (it's a recent recommendation)
```suggestion
from prometheus_client import Counter, Histogram, Gauge
```